pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>com.blueapron</groupId>
     <artifactId>kafka-connect-protobuf-converter</artifactId>
     <packaging>jar</packaging>
-    <version>3.0.1-SNAPSHOT</version>
+    <version>3.1.1-SNAPSHOT</version>
```

Comment:
```suggestion
    <version>3.1.0-SNAPSHOT</version>
```

Reply:
Isn't this supposed to point to the next development version? So 3.1.0 is in maven central now, the next dev version should be 3.1.1-snapshot. Or am I looking at this wrong?